### ISSUE TITLE
Are there any plan to release a native build for ARM - MacOS M1 chip?

### ISSUE THREAD

**Issue by PiercarloSlavazza** [TravaOpenJDK/trava-jdk-11-dcevm#36]

**Comment by markdbuck** [comment]
I wanted to add an ARM build for linux, so I checked travis-ci which is used for building dcevm and unfortunately, it doesn't provide support for arm on macos. So that I think is a limitation in providing this. But, I was able to install dcevm macos version on my m1 macbook air and it seemed to work fine albeit with only a quick check.

**Comment by mic1099** [comment]
@markdbuck yeah installing jdk on apple silicon macs for mac-amd64 works just fine thanks to Rosetta. However you are giving away performance you would be able to achieve running native code on those apple chips. I do need to compile fairly large projects and there is difference between running native jdk versus mac-amd64 one.

**Comment by maiksd** [comment]
I'd like to renew this request, but it looks like travis-ci [still doesn't support ARM on macOS](https://docs.travis-ci.com/user/reference/osx/), or [multiple architectures at all on macOS](https://docs.travis-ci.com/user/multi-cpu-architectures/). This is a bit sad, considering that any new Mac hardware is ARM-based.

**Comment by skybber** [comment]
Travis build has been abandoned for some time, TravaOpenJDK is build by Github Actions now.

**Comment by maiksd** [comment]
Ok, so what's holding the ARM Mac build back then? :-)

**Comment by markdbuck** [comment]
When travis-ci was still being used, I did contribute a build for ARM. But that went by the boards with the move to github actions.
I looked into trying to do ARM build with github actions a while back despite having zero experience with it. I do have lots of CI experience with Jenkins and Tekton, but I found github actions quite confusing. But, I believe I understood enough to find out that github actions doesn't provide support for hosted actions running on ARM. (see, for example, https://github.community/t/ask-about-github-hosted-runners-arm64-platform-support/167483/3). 
I tried looking into uraimo/run-on-arch-action (see https://github.com/uraimo/run-on-arch-action) but quickly got bogged down.

**Comment by asgaralee** [comment]
I am sorry, I  could not understand if DCEVM will be released for ARM version? It is for new M3 silicons by Apple

**Comment by markiewb** [comment]
Checkout the OpenJDK-fork of Jetbrains at https://github.com/JetBrains/JetBrainsRuntime
It includes DCEVM - see the Readme file. 

If one of their Java based development tools like IDEA supports your M3-MacOS platform, then you could use their JDK too.
